### PR TITLE
[libraw] Add include for select_library_configurations (#6715)

### PR DESCRIPTION
**Pull request by trylle**
Please check if this resolves your problem, @mloskot.

### CODE REVIEW

File: ports/libraw/findlibraw_debug_fix.patch [hunk 1]
```diff
@@ -32,6 +32,8 @@
 +             ${PC_LIBRAW_R_LIBRARY_DIRS}
 +            )
 +
++INCLUDE(SelectLibraryConfigurations)
```
Comment (mloskot) [inline ports/libraw/findlibraw_debug_fix.patch]: Shouldn't this line be placed before the line 21?
i.e. before `select_library_configurations(LibRaw)`

Comment (trylle) [reply ports/libraw/findlibraw_debug_fix.patch]: Sorry, should be fixed now.

Comment (mloskot) [reply ports/libraw/findlibraw_debug_fix.patch]: Looks good. Thank you!

**Comment by NancyLi1013** [comment]
Hi @trylle, thanks for your PR. Could you also bump the version in CONTROL file?

File: ports/libraw/CONTROL [hunk 2]
```diff
@@ -1,4 +1,4 @@
 Source: libraw
-Version: 0.19.2
+Version: 0.19.2-1
```
Comment (mloskot) [inline ports/libraw/CONTROL]: This change is no longer necessary due to switch the the snapshot in #6759

Comment (trylle) [reply ports/libraw/CONTROL]: OK, I've rebased the PR onto master and bumped the version to 201903-1 to differentiate from #6759 already merged into master.

**Comment by mloskot** [comment]
Thanks.
What's stopping this from being merged?

**Comment by ras0219-msft** [comment]
Nothing, thanks for the PR!

**Pull request merged**